### PR TITLE
Update csa.md to add note about beta conditions

### PR DESCRIPTION
**Pull request by megankj**

### CODE REVIEW

**Comment by megankj** [comment]
@lannka Quick PR to update the doc page.  Thanks!

File: ads/google/csa.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ limitations under the License.
 -->
 # Custom Search Ads
 
+_The Custom Search Ads integration is offered as a Beta and is subject to the
+conditions of the Google Program Guidelines._
```
Comment (lannka) [inline ads/google/csa.md]: do you have a link about the guidelines?

Comment (mmcao) [inline ads/google/csa.md]: AFS is offered as an Direct and Online product with two sets of GPGs. Can we leave the statement w/o a link so users can interpret it according to their own GPGs? If that's not possible, let's exclude the part referring to the GPGs and just say "The Custom Search Ads integration is offered as a Beta."

Comment (lannka) [inline ads/google/csa.md]: Which ever way you think is better :-) . Let me know if this is good to merge.

Comment (megankj) [inline ads/google/csa.md]: I think it's fine to merge without the link as Mitchell says.  Thanks!

**Pull request merged**